packages/adapters/ethers5/src/tests/client.test.ts

```diff
@@ -11,7 +11,7 @@ import { arbitrum, mainnet, polygon } from '@reown/appkit/networks'
 import { ProviderUtil } from '@reown/appkit/store'
 import { SafeLocalStorage, SafeLocalStorageKeys } from '@reown/appkit-common'
 import { type BlockchainApiLookupEnsName } from '@reown/appkit'
-import { ethers } from 'ethers5'
+import { ethers } from 'ethers'
```

Comment:
Not `ethers5` ?

Reply:
Yeah we don't need it anymore for in this adapter package. When every adapter was in the same package, we needed to separate ethers and ethers v5 with this way:
```json
"ethers5": "npm:ethers@5.7.2",
```

Now since they are separated packages, we don't need it any more. But keep in mind that it's still used for laboratory app since both versions needs to be installed separately.